tensorly/contrib/decomposition/tests/test_tt_TTOI.py

```diff
@@ -0,0 +1,48 @@
+
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Testing of applying TTOI
+
+@author: Lili Zheng
+"""
+
+import numpy as np
+import math
+import tensorly as tl
+from tensorly import random
+import tensorly.decomposition
+from tensorly.contrib.decomposition.tt_TTOI import TTOI 
+
+def test_TTOI(monkeypatch):
+    """Test for the tt_TTOI function (Tensor train orthogonal iteration)
+    """
+    rng = tl.check_random_state(1234)
+    rank = (1, 1, 1, 1, 1, 1)
+    shape = (20, 20, 20, 20, 20)
+    niter = 2
+    
+    # Generate tensor X with low tensor train rank, and its noisy observation Y
+    X = random.random_tt(shape=shape, rank=rank, random_state=rng, full = True)
+    context = tl.context(X)
+    E = tl.tensor(np.random.normal(0,2,size = shape))
```

Comment:
Minor: would be useful to read the code to name variables (E, X, Y...) more clearly.

Reply:
I rename the variables as noise_tensor, true_tensor, and data_tensor. The data_tensor is observed but not of tensor-train low-rank; instead, it equals true_tensor+noise_tensor where true_tensor is of tensor-train low-rank. Our function aims to recover true_tensor from data_tensor. Would these names be clearer?